tools/system_libs.py

```diff
@@ -961,15 +962,19 @@ class libcxx(NoExceptLibrary, MTLibrary):
 class libunwind(NoExceptLibrary, MTLibrary):
   name = 'libunwind'
   cflags = ['-Oz', '-D_LIBUNWIND_DISABLE_VISIBILITY_ANNOTATIONS']
-  src_dir = ['system', 'lib', 'libunwind', 'src']
   # Without this we can't build libunwind since it will pickup the unwind.h
   # that is part of llvm (which is not compatible for some reason).
   includes = [['system', 'lib', 'libunwind', 'include']]
-  src_files = ['Unwind-wasm.cpp']
 
   def __init__(self, **kwargs):
     super(libunwind, self).__init__(**kwargs)
 
+  def get_files(self):
+    return [
+      shared.path_from_root('system', 'lib', 'libunwind', 'src', 'Unwind-wasm.cpp'),
+      shared.path_from_root('system', 'lib', 'compiler-rt', 'lib', 'builtins', 'gcc_personality_v0.c'),
```

Comment:
We don't use gcc_personality_v0.c, and it overlaps with [libcxxabi/src/cxa_personality.cpp](https://github.com/emscripten-core/emscripten/blob/main/system/lib/libcxxabi/src/cxa_personality.cpp), even though they don't look exactly the same. I think we don't need to include this file in libunwind.

Reply:
OK, maybe I'll try just removing it completely.